join-us/index.html

```diff
@@ -11,7 +11,12 @@
         </div>
     </div>
     <div class="row" style="margin-top: 15px; margin-bottom: 30px; text-align: center;">
-        <a href="https://wicsuofm.slack.com"><button class="ui primary button">Join Our Slack Here</button></a>
+        <a href="https://wicsuofm.slack.com">
```

Comment:
Can you please change this page back to the original? I'll include it in another PR. Thanks!

Reply:
Good eye Julia, I will fix that.

Reply:
I think it is good now. Let me know if you need anything else before merging.